week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,14 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  var additionalFee = (1 / 100) * (price * 1.4);
```

Comment:
Really really like how you have got this down to one line in each function. A really neat and simple design!

Reply:
Thank you @mtbdata711. After studying further, I discovered a new way of calculating the price. I have therefore made changes.